WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -763,6 +760,9 @@ public void onTrimMemory(final int level) {
                     break;
                 case TRIM_MEMORY_BACKGROUND:
```

Comment:
Do we want to call `onAppGoesToBackground` in `TRIM_MEMORY_BACKGROUND` as well? I thought we only wanted to do it in `TRIM_MEMORY_UI_HIDDEN`.

Reply:
@aerych Sorry, I couldn't find how to remove this comment before I could submit my review after the code change.